content/githubactions/_index.en.md

```diff
@@ -6,7 +6,4 @@ chapter: true
 
 # Updating Data With GitHub Actions
 
-**This material is currently under development!**
-
-[GitHub Actions](https://github.com/features/actions) is GitHub's built in system that can be used for automating data pipelines. 
-It used to be free for open source projects but now requires payment or applying for open source credits. This tutorial will walk you through setting up an automated data workflow using GitHub Actions + R, git, GitHub, and Zenodo. It assumes a basic knowledge of git and a GitHub account. Either follow the navigation on the left or click the arrow on the right to get started!
+[GitHub Actions](https://github.com/features/actions) is GitHub's built in system that can be used for automating data pipelines. It is our suggested alternative to Travis CI for continuous integration, given that Travis CI is no longer free for open source projects; it now requires payment or applying for open source credits. This tutorial will walk you through setting up an automated data workflow using GitHub Actions + R, git, GitHub, and Zenodo. It assumes a basic knowledge of git and a GitHub account. Either follow the navigation on the left or click the arrow on the right to get started!
```

Comment:
I recommend cutting "It is our suggested alternative to Travis CI for continuous integration, given that Travis CI is no longer free for open source projects; it now requires payment or applying for open source credits." from this file. I think if we want to make recommendations about which of the available options to use we should place that on the landing page with a description of the strengths and weaknesses of each option.